### ISSUE TITLE
how to format the output

### ISSUE THREAD

**Issue by Manjunath-PM** [awslabs/amazon-transcribe-streaming-sdk#53]
aws transcribe is working fine, however the output is like this
This.
This is
This is a
This is a te
This is a test.
This is a test.

I only want the output as
This is a test.

I understand the repeatation line comes as streaming is very fast. how to do streaming so that one word go only once.

What changes need to be done in program simple_file.py orsimple_mic.py.

**Comment by nateprewitt** [comment]
Resolving as a duplicate of #41.
